resources/dynamodb/v1alpha1/global-table.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
this filename should be `resources/dynamodb/v1alpha1/globaltable.yaml` to match the resource name